components/eamxx/src/physics/p3/p3_functions_f90.cpp

```diff
@@ -1685,6 +1685,7 @@ void p3_main_part2_f(
 
   const Int nk = (kte - kts) + 1;
   const Int nk_pack = ekat::npack<Spack>(nk);
+  const Real max_total_ni = 740.0e3;
```

Comment:
Might be worth adding a comment, that this is the value hard-coded in Fortran. The user may be confused as of why this is hard coded, while eamxx has it as a runtime parameter...